### ISSUE TITLE
HTTPS-enable LMS

### ISSUE THREAD

**Issue by Floppy** [theodi/shared#916]

**Comment by Floppy** [comment]
@davetaz Sam said you were going to look at this in Cloudflare rather than Heroku, is that right?

**Comment by davetaz** [comment]
Yes, For now it turned out to be an Internet Exploere problem from the client and not a govenment laptop policy issue.

**Comment by davetaz** [comment]
So this issue is on hold for now.
